src/app/game/game-intermediate-result/game-intermediate-result-header/confirm-exit-dialog/confirm-exit-dialog.component.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../../../../variables.scss';
+
+.cdk-overlay-container .cdk-overlay-pane {
+  border-radius: 1.5rem;
+  overflow: hidden;
+  padding-left: var(--boxPaddingX);
+  padding-right: var(--boxPaddingX);
+
+  width: 40vw;
+  height: 30vh;
+}
+
+#exitGameText {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: inherit;
+  height: 10vh;
+  font-size: var(--fontSizeSmall); // make to variable

```

Comment:
I know you have commented on the styles already, but maybe change the font size to be a little bit bigger? In case we forget to edit this later

Reply:
Problem with this is that when the correct css is applied, this font size is pretty big. So the one we see in the screenshot is the hidden CSS that we struggle with here and in idletimeout

Reply:
I found that it helps to remove the mat-raised-button class from the html tag. This is probably what causes the problems in idle-timeout.component as well